extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/CompassWidget.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.maps.extension.androidauto
+
+import android.content.Context
+import android.graphics.BitmapFactory
+import com.mapbox.maps.renderer.widget.BitmapWidget
+import com.mapbox.maps.renderer.widget.WidgetPosition
+
+class CompassWidget(context: Context) : BitmapWidget(
+  bitmap = BitmapFactory.decodeResource(context.resources, R.drawable.mapbox_compass_icon),
+  position = WidgetPosition(
+    horizontal = WidgetPosition.Horizontal.LEFT,
+    vertical = WidgetPosition.Vertical.CENTER,
+  ),
+  marginX = 20f,
```

Comment:
should we give opportunity for user to change this value? 
and what about different screen resolutions? should we add some scale factor?

Reply:
> should we give opportunity for user to change this value?
Good catch, I'll add positioning properties to both widget impls. 

> and what about different screen resolutions? should we add some scale factor?
Could be easily implemented user-side, values here are all in pixels, user could use dp and scale them to pixels. I'd kept the API as simple as possible for the first iteration